src/celestia/macosx/BrowserItem.mm

```diff
@@ -80,7 +80,7 @@ + (void)addChildrenToStar: (BrowserItem *) aStar
         BrowserItem *minorMoons = nil;
         BrowserItem *asteroids = nil;
         BrowserItem *comets = nil;
-        BrowserItem *spacecrafts = nil;
+        BrowserItem *spacecraft = nil;
```

Comment:
cocoa frontend is not maintained for 1.7.0, so just leave it unchanged

Reply:
Since I already changed it I see no need to revert just because the frontend isn't maintained.